src/module/combat/combat.js

```diff
@@ -553,9 +553,13 @@ export class CombatSFRPG extends Combat {
 
     _getInitiativeFormula(combatant) {
         if (this.getCombatType() === "starship") {
-            return "1d20 + @pilot.skills.pil.mod"
-        }
-        else {
+            if (combatant.actor.data.data.crew.useNPCCrew) {
+                return "1d20 + @pilot.skills.pil.mod + @combatant.attributes.systems.engines.mod + @combatant.attributes.systems.powerCore.modOther + @additional.modifiers.bonus";
```

Comment:
```suggestion
                return "1d20 + @pilot.skills.pil.mod + @additional.modifiers.bonus";
```

Reply:
That's because `@combatant.attributes.systems.engines.mod + @combatant.attributes.systems.powerCore.modOther` are already included in `@pilot.skills.pil.mod` of an NPC.

Reply:
This is what I explained to you in the Discord. `@combatant.attributes.systems.engines.mod + @combatant.attributes.systems.powerCore.modOther` are for system damage; if an NPC has a damaged engine or power core, they'll take a penalty to their initiative. This isn't included in the formula as it's dynamic. `@combatant.attributes.pilotingBonus.value` is the bonus from thrusters/maneuverability, which ***is*** included in the NPC's piloting mod so has been omitted from that line there.